emails/utils.py

```diff
@@ -275,3 +275,37 @@ def decrypt_reply_metadata(key, jwe):
     e.deserialize(jwe)
     e.decrypt(k)
     return e.plaintext
+
+
+class S3ClientException(Exception):
+    """Exception raised by error on S3 Client using Boto3.
+
+    Attributes:
+        message -- optional explanation of the error
+    """
+
+    def __init__(self, message=None):
+        self.message = message
+
+
+def get_message_content_from_s3(bucket, object_key):
+    try:
+        emails_config = apps.get_app_config('emails')
+        streamed_s3_object = emails_config.s3_client.get_object(
```

Comment:
Not sure if this looks any cleaner to skip the `emails_config` var and go straight to `s3_client`?
```suggestion
        s3_client = apps.get_app_config('emails').s3_client
        streamed_s3_object = s3_client.get_object(
```